Jenkinsfile_CNP

```diff
@@ -141,15 +141,15 @@ withPipeline(type, product, component) {
     env.SKIP_FUNCTIONAL_TESTS = 'true'
   }
 
-  before('functionalTest:aat') {
-    env.SKIP_FUNCTIONAL_TESTS = 'true'
-  }
-
   before('functionalTest:demo') {
     env.SKIP_FUNCTIONAL_TESTS = 'true'
   }
 
   after('functionalTest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
   }
+
+  after('functionalTest:aat') {
```

Comment:
Please just move it before `after('functionalTest:preview') {` (one position up).

Reply:
Moved 'after('functionalTest:aat')' one position up before 'after('functionalTest:preview')'